src/fetch/payouts.py

```diff
@@ -22,8 +23,10 @@
 from src.pg_client import MultiInstanceDBFetcher
 from src.utils.print_store import Category
 
-PERIOD_BUDGET_COW = 306646 * 10**18
-QUOTE_REWARD = 9 * 10**18
+PERIOD_BUDGET_COW = 250000 * 10**18
+CONSISTENCY_REWARD_CAP = 6 * 10**18
```

Comment:
let's explicitly add the token denomination in the name of the variable here.

E.g., `CONSISTENCY_REWARD_ETH_CAP`

Reply:
I added the currency at the end. This is a convention we use in some of the queries.